Orange/canvas/mainwindow.py

```diff
@@ -1,7 +1,51 @@
+from AnyQt.QtCore import Qt
+from AnyQt.QtWidgets import (
+    QFormLayout, QCheckBox, QLineEdit, QWidget, QVBoxLayout,
+)
+from orangecanvas.application.settings import UserSettingsDialog
 from orangewidget.workflow.mainwindow import OWCanvasMainWindow
+
 from Orange.canvas.utils.overlay import NotificationOverlay
 
 
+class OUserSettingsDialog(UserSettingsDialog):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        w = self.widget(0)  # 'General' tab
+        layout = w.layout()
+        assert isinstance(layout, QFormLayout)
+        cb = QCheckBox(self.tr("Automatically check for updates"))
+        cb.setAttribute(Qt.WA_LayoutUsesWidgetRect)
+
+        layout.addRow("Updates", cb)
+        self.bind(cb, "checked", "startup/check-updates")
+
+        # Error Reporting Tab
+        tab = QWidget()
+        self.addTab(tab, self.tr("Error Reporting"),
+                    toolTip="Settings related to error reporting")
+
+        form = QFormLayout()
+        line_edit_mid = QLineEdit()
+        self.bind(line_edit_mid, "text", "error-reporting/machine-id")
+        form.addRow("Machine ID:", line_edit_mid)
+
+        box = QWidget()
+        layout = QVBoxLayout()
+        layout.setContentsMargins(0, 0, 0, 0)
+        cb1 = QCheckBox(
+            self.tr(""),
+            toolTip=self.tr(
+                "Share anonymous usage statistics to improve Orange")
+        )
+        self.bind(cb1, "checked", "error-reporting/send-statistics")
```

Comment:
Statistics should be enabled/disabled dynamically upon preference change, similar to when the notification is responded to.
```
cb1.clicked.connect(lambda checked:
                    UsageStatistics.set_enabled(checked))
```